htdocs/document.php

```diff
@@ -218,7 +219,9 @@ function llxFooter()
 $check_access = dol_check_secure_access_document($modulepart, $original_file, $entity, $user, $refname);
 $accessallowed              = $check_access['accessallowed'];
 $sqlprotectagainstexternals = $check_access['sqlprotectagainstexternals'];
-$fullpath_original_file     = $check_access['original_file']; // $fullpath_original_file is now a full path name
+if (empty($fullpath_original_file)) {
+	$fullpath_original_file     = $check_access['original_file'];
```

Comment:
$check_access['original_file']  is a path on server side.
$ecmfile->fullpath_orig  is the name that the file has on local computer before uploading it. 

There is no relation between both so i see no reason to have an affect on 2 var that has different meanings.